pkg/ginkeycloak/gin_keycloak.go

```
@@ -60,7 +63,9 @@ func GetTokenContainer(token *oauth2.Token, config KeycloakConfig) (*TokenContai
 func getPublicKey(keyId string, config KeycloakConfig) (string, string, error) {
 	keyEntry, exists := publicKeyCache.Get(keyId)
 	if !exists {
-		url := config.Url + "/auth/realms/" + config.Realm + "/protocol/openid-connect/certs"
+		u, _ := url.Parse(config.Url)
```

you should not ignore the err. pleas return err if not nil